Fixes typos and improves clarity of [DEV] Perf Tools for VS Code Development (#306)
microsoft/vscode-wiki: [DEV]-Perf-Tools-for-VS-Code-Development.md

The page lists some tools to investigate and understand the performance of VS Code. They complement the built-in Performance, CPU Profiler, and Memory tabs of dev tools and usually require code changes (which usually must not be committed).

The following tools are available to the "core" of VS Code, e.g main, renderer, shared, and extension host process but not to extensions. 

### Performance View/Editor

> The startup performance editor (F1 > Developer: Startup Performance) gives you a good overview of things that are important for startup, e.g

Reviewer: ```suggestion
The startup performance editor (F1 > Developer: Startup Performance) gives you a good overview of things that are important for startup, such as
```
